spyder/widgets/collectionseditor.py

```diff
@@ -1879,7 +1879,7 @@ def __init__(self):
             'ddataframe': test_df,
             'None': None,
             'unsupported1': np.arccos,
-            'unsupported2': np.cast,
+            'unsupported2': np.asarray,
```

Comment:
Is this going to crash users on import if they have spyder + numpy2 installed?

i wonder if it is worth updating the constraints on conda-forge to make spyder incompatible with numpy 2.

Unlike at pypi, we are in a position to do this.

Reply:
> Is this going to crash users on import if they have spyder + numpy2 installed?

Nop, this a simple test at the of the file to check that the widget defined on it is working as expected. It was the way we used to test Spyder before adopting `pytest` and we decided to leave them because they catch problem from time to time.

> i wonder if it is worth updating the constraints on conda-forge to make spyder incompatible with numpy 2.

I don't think there's a need to do that because Spyder can work with Numpy 1 and 2 arrays without issues.

Reply:
thank you for confirming